models/place.js

```diff
@@ -0,0 +1,16 @@
+const mongoose = require('mongoose');
+const Schema = mongoose.Schema;
+// mongoose.connect('mongodb://localhost/ibi-ironhack',  {useMongoClient: true});
+// mongoose.Promise = require('bluebird');
+
+const placeSchema = new Schema({
+    name: {type: String},
+    location: {
+                lat: {type: String},
+                lng: {type: String},
+            },
+    type: {type: String}
+});
+
```

Comment:
Again with coordinates, your queries are going to improve A LOT if you create a geospatial index, g.e.:

```javascript
placeSchema.index({location: '2dsphere'});
```